src/components/Game/Game.js

```diff
@@ -56,6 +57,30 @@ class GameView extends Component {
         window.removeEventListener('resize', this.debouncedResizeListener);
     }
 
+    handleResize = () => {
+        const { letters, lettersInSlots, dragDisabled } = this.state;
+        if (isMobile() === dragDisabled) {
+            return;
+        }
+        if (!isMobile()) {
+            const lettersOutsideSlots = letters
+                .filter((letter) => letter.active);
+
+            this.setState({
+                letters: lettersInSlots.concat(lettersOutsideSlots),
+                currentAnswer: [...lettersInSlots.concat(lettersOutsideSlots)],
```

Comment:
I wonder if we can simplify the state by getting rid of `letterInSlots`
given you introduced `currentAnswer` I sense `letterInSlots` does not provided any added value, it basically holds the same information, doesn't it? and it's always better to have as little elements as possible

btw, lines 70 and 71 contains the same logic (concat), perhaps it can be written once few lines above

Reply:
right! :)))